disperser/cmd/batcher/flags/flags.go

```diff
@@ -36,16 +36,22 @@ var (
 		Required: true,
 		EnvVar:   common.PrefixEnvVar(envVarPrefix, "PULL_INTERVAL"),
 	}
+	AddressDirectoryFlag = cli.StringFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "address-directory"),
+		Usage:    "Address of the EigenDA Address Directory (preferred over individual contract addresses)",
+		Required: false,
+		EnvVar:   common.PrefixEnvVar(envVarPrefix, "ADDRESS_DIRECTORY"),
+	}
 	BlsOperatorStateRetrieverFlag = cli.StringFlag{
 		Name:     common.PrefixFlag(FlagPrefix, "bls-operator-state-retriever"),
-		Usage:    "Address of the BLS Operator State Retriever",
-		Required: true,
-		EnvVar:   common.PrefixEnvVar(envVarPrefix, "BLS_OPERATOR_STATE_RETRIVER"),
+		Usage:    "Address of the BLS Operator State Retriever (legacy - use address-directory instead)",
+		Required: false,
+		EnvVar:   common.PrefixEnvVar(envVarPrefix, "BLS_OPERATOR_STATE_RETRIEVER"),
```

Comment:
I changed all the `BLS_OPERATOR_STATE_RETRIVER` to `BLS_OPERATOR_STATE_RETRIEVER` bc the former has a typo, and might affect cluster env var. for vis: @dmanc

Reply:
Can we keep the old name since I assume we would deprecate this config at some point?